src/Akka.Cluster.Hosting/AkkaClusterHostingExtensions.cs

```diff
@@ -824,7 +824,7 @@ internal static Config CreateReceptionistConfig(string name, string? role)
                 .Append(root).Append("name:").AppendLine(name.ToHocon());
             
             if(!string.IsNullOrEmpty(role))
-                sb.Append(root).Append("role:").AppendLine(role.ToHocon());
+                sb.Append(root).Append("role:").AppendLine(role!.ToHocon());
```

Comment:
LGTM - `ToHocon` has `null` handling built into it, IIRC? @Arkatufus

Reply:
Its compiler/roslyn error, its already checked for null by the line before it.